test/utils/addressFactory.js

```diff
@@ -1,25 +1,22 @@
 function addressFactory(quantity) {
-  const addresses = [];
-  for (let i = 0; i < quantity; i++) {
-    const address = {
-      streetName: `${i} street`,
-      addressLocality: `${i} town`,
-      postalCode: `${i} code`,
+  const addresses = [
+    {
+      subBuildingName: "flat 1",
+      buildingNumber: "1",
+      streetName: "street1",
+      addressLocality: "town1",
+      postalCode: "postcode1",
       validFrom: String(new Date().getFullYear()),
-    };
+    },
+    {
+      buildingName: "farm2",
+      addressLocality: "town2",
+      postalCode: "postcode2",
+      validFrom: String(new Date().getFullYear()),
+    },
+  ];
 
-    // factor in when an address has building name or building number or both
-    if (i % 3 === 0) {
-      address.buildingNumber = `${i}`;
-    } else if (i % 3 === 1) {
-      address.buildingName = `${i} house`;
-    } else {
-      address.buildingNumber = `${i}`;
-      address.buildingName = `Manor house`;
-    }
-    addresses.push(address);
-  }
-  return addresses;
+  return addresses.slice(0, quantity);
```

Comment:
Its good that this change didnt break anything else. 

If we want to add more than two addresses in the future, this factory will need to be relooked at.
The logic before was applying different style addresses based on which position in a list they exist.